contracts/ownership/NoOwner.sol

```diff
@@ -11,5 +11,5 @@ import "./HasNoContracts.sol";
  * @dev Solves a class of errors where a contract accidentally becomes owner of Ether, Tokens or
  * Owned contracts. See respective base contracts for details.
  */
-contract NoOwner is HasNoEther, HasNoTokens, HasNoContracts {
+contract NoOwner is HasNoEther, HasNoTokens, HasNoContracts { // solium-disable-line no-empty-blocks
```

Comment:
Is the `no-empty-blocks` rule ever useful?

Reply:
honestly I don't know. We have a lot of them because of the open implementations. I'll disable to project-wide and remove the inline comments unless we can think of a really good reason not to.